commit 9df1b480eb008c63d6dab167fc992e172730ba77
Author: Martin Cermak <mcermak@redhat.com>
Date:   Thu Sep 22 15:10:29 2016 +0200

    Fix rhbz1378462 by updating the path to stap-gen-cert in stap-server.

diff --git a/stap-server b/stap-server
index c39ae49..7cb75d6 100644
--- a/stap-server
+++ b/stap-server
@@ -503,7 +503,7 @@ prepare_stat_dir () {
 prepare_certs () {
     if [ "$USER" != "`id -un`" ]; then
         if ! runuser -s /bin/bash - $USER -c 'test -f $HOME/.systemtap/ssl/server/stap.cert'; then
-            runuser -s /bin/bash - $USER -c %{_libexecdir}/systemtap/stap-gen-cert >/dev/null
+            runuser -s /bin/bash - $USER -c ${PKGLIBEXECDIR}stap-gen-cert >/dev/null
         fi
     else
         if ! test -f $HOME/.systemtap/ssl/server/stap.cert; then
